src/components/Description.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Alert} from '@mui/material';
+
+const Description = () => {
+  return (
+    <Alert severity="info" className='app-desc'>
+      This app get NASA's Astronomy Picture of the Day (APOD)
+       and text it to users' phones everyday.
```

Comment:
watch for typos.. this app get**s**
text**s** it

Also, you should write for the user, not about them. Pretend we want people using this

"Get NASA's Astronomy Picture of the Day (APOD) in a daily text"